lib/utils.js

```diff
@@ -179,6 +180,21 @@ function batteryVoltageToPercentage(voltage, option) {
     } else if (option === 'Add_1V_42V_CSM300z2v2') {
         voltage = voltage + 1000;
         percentage = toPercentage(voltage, 2900, 4100);
+    // Generic converter that expects an option value to be given in the format of:
+    // GENERIC_%MinimumThresholdVoltage%_%MaximumThresholdVoltage%
+    // I.E. GENERIC_1900_3000
+    } else if (optionSplit[0] === 'GENERIC') {
```

Comment:
I think it is better to make this an object.

So:

- `meta: {battery: {voltageToPercentage: {min: 1900, max: 3000}}},`
- `else if (typeof option === 'object') {`
- `percentage = toPercentage(voltage, option.min, option.max);`

Reply:
Was thinking of using that approach so will swap it out.

Thanks!

Reply:
Done!